README.md

```diff
@@ -1,19 +1,23 @@
 # cffconvert GitHub Action
 
-GitHub action to validate CITATION.cff files, and convert to other citation formats.
+GitHub action to validate CITATION.cff files, and convert to other citation formats using [cffconvert](https://github.com/citation-file-format/cff-converter-python).
```

Comment:
I prefer to link to pypi, and maybe change the text to include "dockerized version of cffconvert"

Reply:
done